keyserver/testutil/testutil.go

```diff
@@ -0,0 +1,113 @@
+package testutil
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"io/ioutil"
+	"math/big"
+	"net"
+	"os"
+	"path"
+	"testing"
+	"time"
+)
+
+const (
+	TestDir          = "coniksServerTest"
+	PublicConnection = "127.0.0.1:3000"
+	LocalConnection  = "127.0.0.1:3001"
+)
+
+func CreateTLSCert(t *testing.T) (dir string, teardown func()) {
```

Comment:
Nitpicking: If you give the return value a name (like here), I think it would be good to actually use it below (like you did with `dir`)